### PR TITLE
(MODULES-4325) README edit for loc

### PR DESCRIPTION
**Pull request by jbondpdx**
This one was pretty straightforward, though I moved a lot around.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,44 +11,69 @@
 5. [Development - Guide for contributing to the module](#development)
 
 
-## Module Description
+## Module description
```
Comment (bmjen) [inline README.md]: Module README Template shows this as

`## Description`

**Comment by jbondpdx** [comment]
Fixed, @bmjen !

File: README.md [hunk 2]
```diff
@@ -11,44 +11,69 @@
...
-        report = true
-        pluginsync = true
+  ```
+[agent]
+report = true
+pluginsync = true
+  ```
 
-2. On the Puppet master, make sure the [`reports`](https://docs.puppetlabs.com/references/4.2.latest/configuration.html#reports) setting in the master section includes tagmail:
+2. On the Puppet master, include tagmail in the  [`reports`](https://docs.puppetlabs.com/references/4.2.latest/configuration.html#reports) setting in the master section:
```
Comment (eputnam) [inline README.md]: extra space before the first open bracket?

File: README.md [hunk 3]
```diff
@@ -11,44 +11,69 @@
...
+To configure the tagmail module, edit the `tagmail.conf` file you created in Step 4 above. This file is located in your Puppet confdir. The `tagmail.conf` should be formatted as an ini file.
+
+1. Open `tagmail.conf` in your text editor and add create `[transport]` and `[tagmap]` sections.
+
+1. In the `[transport]` section, specify _either_:
+
+  * `sendmail`, with a path to your sendmail binary (by default, `/usr/sbin/sendmail`).
+  * `smtpserver`, `smtpport`, and `smtphelo`. If you do not specify `smtpserver`, tagmail defaults to using `sendmail`.
+
+1. In the `[tagmap]` section , specify tags and email addresses. Each line should include both:
```
Comment (eputnam) [inline README.md]: numbering is weird in the above section. the raw md says 1. 1. 1. but when i look at the rich preview its 1. 2. 1.

File: README.md [hunk 4]
```diff
@@ -58,38 +83,28 @@ smtphelo = example.org
...
-
-The above example sends all log messages to `me@example.com`, and all messages from webservers that are not also from mailservers to `httpadmins@example.com` and to `you@example.com`.
-
+```
 
 ## Limitations
 
-This module supports Puppet Enterprise and Puppet versions 3.8 or newer. For older versions of Puppet, use Puppet's built-in tagmail feature.
-
+This module should be used only with Puppet Enterprise and Puppet versions 3.7 or newer, and only if you're using the JVM on the Puppet master. For older versions of Puppet, or if using the legacy Puppet master on Apache/Rack/Passenger, use Puppet's built-in tagmail feature.
```
Comment (eputnam) [inline README.md]: In the requirements section we say Puppet 3.8 as the lower bound and i think that's what it should be here too.

**Comment by jbondpdx** [comment]
@eputnam those issues should all be fixed now. Good catches! Let me know if you want me to squash before merge.

File: README.md [hunk 5]
```diff
@@ -2,7 +2,7 @@
 
 #### Table of Contents
 
-1. [Module Description - What the module does and why it is useful](#module-description)
+1. [Module description - What the module does and why it is useful](#module-description)
```
Comment (eputnam) [inline README.md]: One last thing! I think we need to at least update the id in this link to reflect the change to the description heading.

Comment (jbondpdx) [inline README.md]: Oof! Yeah we do!

**Pull request merged**